+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Thu Jun 22 09:16:28 2023 - INFO  - Start to prepare the test environment.
+ OLD_LANG=en_US.UTF-8
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ DNF_INSTALL 'bacula-client mysql5-server bacula-common tar'
+ pkgs='bacula-client mysql5-server bacula-common tar'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs 'bacula-client mysql5-server bacula-common tar' --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 0:07:10 ago on Thu 22 Jun 2023 09:09:38 AM CST.
No match for argument: mysql5-server
Package tar-2:1.34-4.oe2303.riscv64 is already installed.
Error: Unable to find a match: mysql5-server'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 0:07:10 ago on Thu 22 Jun 2023 09:09:38 AM CST.
No match for argument: mysql5-server
Package tar-2:1.34-4.oe2303.riscv64 is already installed.
Error: Unable to find a match: mysql5-server'
+ tar -zxvf common/test.tar.gz
config/
config/a.config
config/4.conf
config/3.conf
config/2.conf
config/6.conf
config/5.conf
config/1.conf
config/e.conf
+ systemctl restart mysqld
Failed to restart mysqld.service: Unit mysqld.service not found.
+ mkdir tmp
+ /usr/libexec/bacula/create_mysql_database
oe_test_bacula_bconsole_02.sh: line 30: /usr/libexec/bacula/create_mysql_database: No such file or directory
+ /usr/libexec/bacula/make_mysql_tables
oe_test_bacula_bconsole_02.sh: line 31: /usr/libexec/bacula/make_mysql_tables: No such file or directory
+ alternatives --set libbaccats.so /usr/lib64/libbaccats-mysql.so
cannot access /var/lib/alternatives/libbaccats.so: No such file or directory
+ sed -i 's\dbuser = "bacula"\dbuser = "root"\g' /etc/bacula/bacula-dir.conf
sed: can't read /etc/bacula/bacula-dir.conf: No such file or directory
+ systemctl start bacula-dir.service
Failed to start bacula-dir.service: Unit bacula-dir.service not found.
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Thu Jun 22 09:16:50 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Jun 22 09:16:51 2023 - INFO  - Start to run test.
+ expect
parent: waiting for sync byte
parent: telling child to go ahead
parent: now unsynchronized from child
couldn't execute "bconsole": no such file or directory
    while executing
"spawn bconsole -u 30"
+ grep '1000 OK' tmp/1.txt
+ CHECK_RESULT 1 0 0 'test bconsole -u failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='test bconsole -u failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'test bconsole -u failed'
+ LOG_ERROR 'test bconsole -u failed'
+ message='test bconsole -u failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'test bconsole -u failed'
Thu Jun 22 09:16:52 2023 - ERROR - test bconsole -u failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_bacula_bconsole_02.sh line 47'
+ message='oe_test_bacula_bconsole_02.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_bacula_bconsole_02.sh line 47'
Thu Jun 22 09:16:52 2023 - ERROR - oe_test_bacula_bconsole_02.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ bconsole -t
oe_test_bacula_bconsole_02.sh: line 48: bconsole: command not found
+ CHECK_RESULT 127 0 0 'test bconsole -t failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='test bconsole -t failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'test bconsole -t failed'
+ LOG_ERROR 'test bconsole -t failed'
+ message='test bconsole -t failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'test bconsole -t failed'
Thu Jun 22 09:16:53 2023 - ERROR - test bconsole -t failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_bacula_bconsole_02.sh line 49'
+ message='oe_test_bacula_bconsole_02.sh line 49'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_bacula_bconsole_02.sh line 49'
Thu Jun 22 09:16:54 2023 - ERROR - oe_test_bacula_bconsole_02.sh line 49
+ '[' 0 -eq 1 ']'
+ return 0
+ grep 'Usage: '
+ bconsole '-?'
+ CHECK_RESULT 1 0 0 'test bconsole -? failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='test bconsole -? failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'test bconsole -? failed'
+ LOG_ERROR 'test bconsole -? failed'
+ message='test bconsole -? failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'test bconsole -? failed'
Thu Jun 22 09:16:54 2023 - ERROR - test bconsole -? failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_bacula_bconsole_02.sh line 51'
+ message='oe_test_bacula_bconsole_02.sh line 51'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_bacula_bconsole_02.sh line 51'
Thu Jun 22 09:16:55 2023 - ERROR - oe_test_bacula_bconsole_02.sh line 51
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Jun 22 09:16:56 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 3
+ test 3 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Jun 22 09:16:56 2023 - ERROR - failed to execute the case.
+ exit 3
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Jun 22 09:16:57 2023 - INFO  - Start to restore the test environment.
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ systemctl stop bacula-dir.service
Failed to stop bacula-dir.service: Unit bacula-dir.service not loaded.
+ sed -i 's\dbuser = "root"\dbuser = "bacula"\g' /etc/bacula/bacula-dir.conf
sed: can't read /etc/bacula/bacula-dir.conf: No such file or directory
+ alternatives --set libbaccats.so /usr/lib64/libbaccats-postgresql.so
cannot access /var/lib/alternatives/libbaccats.so: No such file or directory
+ /usr/libexec/bacula/drop_mysql_tables
oe_test_bacula_bconsole_02.sh: line 61: /usr/libexec/bacula/drop_mysql_tables: No such file or directory
+ /usr/libexec/bacula/drop_mysql_database
oe_test_bacula_bconsole_02.sh: line 62: /usr/libexec/bacula/drop_mysql_database: No such file or directory
+ systemctl stop mysqld
Failed to stop mysqld.service: Unit mysqld.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 0:07:10 ago on Thu 22 Jun 2023 09:09:38 AM CST.
No match for argument: mysql5-server
Package tar-2:1.34-4.oe2303.riscv64 is already installed.
Error: Unable to find a match: mysql5-server ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 0:07:10 ago on Thu 22 Jun 2023 09:09:38 AM CST.
No match for argument: mysql5-server
Package tar-2:1.34-4.oe2303.riscv64 is already installed.
Error: Unable to find a match: mysql5-server'
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'Last metadata expiration check: 0:07:10 ago on Thu 22 Jun 2023 09:09:38 AM CST.\nNo match for argument: mysql5-server\nPackage tar-2:1.34-4.oe2303.riscv64 is already installed.\nError: Unable to find a match: mysql5-server'
+ '[' 0 -ne 0 ']'
+ rm -rf config/ '/var/lib/mysql/*' tmp/ '/var/spool/bacula/*'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Jun 22 09:17:00 2023 - INFO  - End to restore the test environment.
